packages/platform-ui/vitest.config.ts

```
@@ -6,6 +6,15 @@ export default defineConfig({
     environment: 'jsdom',
     setupFiles: ['./vitest.setup.ts'],
     globals: true,
+    include: ['__tests__/**/*.test.{ts,tsx}', 'src/**/*.{test,spec}.{ts,tsx}', 'e2e/**/*.{test,spec}.{ts,tsx}'],
```

[minor] Thanks for adding the e2e glob. Instead of excluding all of the builder/config view suites here, please delete those tests alongside the components you removed. Leaving them in the tree but excluded means they still import non-existent modules (they already fail to compile if run directly) and it silently hides whether new code starts depending on those directories again. Removing the files keeps the repo honest and avoids test coverage quietly rotting behind .